setup.py

```diff
@@ -26,6 +27,11 @@
 # dev_dpmjetIII193=/full/path/to/dir/dpmjetIII-19.3
 # ----
 
+# Set environment variable VIRTUAL_ENV to venv directory
+# It is required in FindPython to find a correct version of python
+# when venv is used in cibuildwheels
+os.environ["VIRTUAL_ENV"] = str(Path(sys.executable).absolute().parents[1])
```

Comment:
This seems like a workaround for buggy detection in find_package(Python ...). Please add the comment

```suggestion
# Set environment variable VIRTUAL_ENV to venv directory
# It is required in FindPython to find a correct version of python
# when venv is used in cibuildwheels, this is a workaround and should be removed
# when FindPython is fixed upstream in the future.
os.environ["VIRTUAL_ENV"] = str(Path(sys.executable).absolute().parents[1])
```

Reply:
As I described the problem about it is a problem of `cibuildwheels`. I will write this explicitly in comments.